assesment.css

```diff
@@ -0,0 +1,37 @@
+body{
+    background-color: lavender;
+    display: grid;
+    grid-template-columns: 50%;
+    justify-content: center;
+}
+
+.header{
+    width: 70%;
+    height: 150px;
+    margin-bottom: 30px;
+    padding: 10px;
+    background-color: skyblue;
+    background-size: inherit;
+    display: table;
```

Comment:
I see you were experimenting with `display: table` and `table-cell`.  However, `display: flex` would have been easier here. I would prevent you from having to specify the columns that the header and image must span. You can talk to @lakerfan1994 (Chuck) to see how he approached this and what to do for our website to be more responsive.